tools/WinObjC.Tools/vsimporter-templates/WinStore10-App/project.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This file can be removed since it is a duplicate of vsimporter-templates/project.json